public/js/priceChart.js

```diff
@@ -0,0 +1,315 @@
+var dataSets = [];
```

Comment:
I see you've installed chart.js as a dependancy to the node project. this means we can just require it in the priceChart js file.

Reply:
priceChart.js file si purely all my JS functions - so I did not add it.